packages/manager/modules/sms/src/sms/batches/details/delete/routing.js

```diff
@@ -0,0 +1,26 @@
+import component from './telecom-sms-batches-history-details-delete.component';
+
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('sms.service.batches.details.delete', {
+    params: {
```

Comment:
No `url` ?

Reply:
Yes, I choose to not have this state accessible by reloading or with bookmark, as it is a modal to delete an item

Reply:
Imo, we're losing part of the benefits of having a state for a modal if we can't access it with a url

Reply:
> Yes, I choose to not have this state accessible by reloading or with bookmark, as it is a modal to delete an item

The modal warn the customer about the deletion and ask a confirmation, so IMHO it's ok to be able to reach this modal directly. 
It's already the cases for some deletion modals in the repository : 
https://github.com/ovh/manager/blob/master/packages/manager/modules/pci/src/projects/project/instances/delete/delete.routing.js#L2-L3

Reply:
> The modal warn the customer about the deletion and ask a confirmation, so IMHO it's ok to be able to reach this modal directly.

In fact the modal state parameter `outgoing` is send by an action to be done by its parent state, not through a resolve. So it is strongly binded to it.
Adding a url to access to this modal would breaks the required `outgoing` param.

(See https://github.com/ovh/manager/pull/3692/files#diff-c54c173aa31a43fe8e6f5fc0fa9bf84eR102 + https://github.com/ovh/manager/pull/3692/files#diff-048f72c5252514d9d1d68957cdae4a18R40)